docs/user/howto_config.md

```diff
@@ -48,6 +53,11 @@ In case `config.yaml` is not provided, the following default settings will be us
 ```yaml
 apiServer:
     advertiseAddress: ""
+    auditLog:
+        maxFileAge: 0
+        maxFileSize: 0
+        maxFiles: 0
+        profile: ""
```

Comment:
This doesn't look like the default values we use. Have you tried with `// +kubebuilder:default=VALUE`?